src/balatro_gym/cards/interfaces.py

```diff
@@ -28,28 +28,53 @@ class Suit(Enum):
 class RankVal:
     """An annoying workaround to allow using an Enum since aenum doesn't have good typing support."""
     _val: int
+    _order: int
 
-    def __init__(self, val: int) -> None:
+    def __init__(self, val: int, order: int) -> None:
         self._val = val
+        self._order = order
 
     @property
     def value(self) -> int:
         return self._val
 
+    @property
+    def order(self) -> int:
+        return self._order
+
 class Rank(Enum):
-    ACE = RankVal(11)
-    KING = RankVal(10)
-    QUEEN = RankVal(10)
-    JACK = RankVal(10)
-    TEN = RankVal(10)
-    NINE = RankVal(9)
-    EIGHT = RankVal(8)
-    SEVEN = RankVal(7)
-    SIX = RankVal(6)
-    FIVE = RankVal(5)
-    FOUR = RankVal(4)
-    THREE = RankVal(3)
-    TWO = RankVal(2)
+    ACE = RankVal(11, 1)
+    KING = RankVal(10, 13)
+    QUEEN = RankVal(10, 12)
+    JACK = RankVal(10, 11)
+    TEN = RankVal(10, 10)
+    NINE = RankVal(9, 9)
+    EIGHT = RankVal(8, 8)
+    SEVEN = RankVal(7, 7)
+    SIX = RankVal(6, 6)
+    FIVE = RankVal(5, 5)
+    FOUR = RankVal(4, 4)
+    THREE = RankVal(3, 3)
+    TWO = RankVal(2, 2)
+
+    @staticmethod
+    def from_int(int_rank: int) -> "Rank":
```

Comment:
If we make `Rank` an `IntEnum` maybe we can eliminate this map?

Reply:
If we do that, I think the values associated to the eval needs to be only an int, so we won't be able to store the value of the rank, just the ordering

```
class Rank(enum.IntEnum):
    ACE = 1 # We don't have the 11 value, same for the face cards
```

Reply:
Yes, indeed. I had it reversed